model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -926,6 +927,23 @@ class JslDslValidator extends AbstractJslDslValidator {
             }
         }
     }
+    
+    @Check
+    def checkAbstractComposition(EntityMemberDeclaration member) {
+        if (member instanceof EntityFieldDeclaration) {
```

Comment:
Wouldn't it be more efficient if the definition parameter would be `EntityFieldDeclaration`?